blog/_posts/2017-09-18-story-of-an-issue.md

```diff
@@ -0,0 +1,100 @@
+---
+title: A story of an issue  
```

Comment:
Hi @netagonen , thanks for your 2nd blog post PR. Please check the comments below.